tests/by-util/test_cksum.rs

```diff
@@ -663,3 +663,195 @@ fn test_conflicting_options() {
         )
         .code_is(1);
 }
+
+#[test]
+fn test_check_algo_err() {
+    let scene = TestScenario::new(util_name!());
+
+    let at = &scene.fixtures;
+
+    at.touch("f");
+
+    scene
+        .ucmd()
+        .arg("--a")
+        .arg("sm3")
+        .arg("--check")
+        .arg("f")
+        .fails()
+        .no_stdout()
+        .stderr_contains("cksum: no properly formatted checksum lines found")
```

Comment:
Is it on purpose that the error message is slightly different from GNU `cksum`? It shows the file name, too:
```
$ cksum -a sm3 --check f
cksum: f: no properly formatted checksum lines found
```

Reply:
Related, I noticed that GNU `cksum` shows an error for each invalid file whereas uutils `cksum` shows a single error.

Reply:
just an oversight from me :)
well spotted!